app/src/main/java/org/oppia/android/app/profile/PinPasswordActivityPresenter.kt

```diff
@@ -51,8 +51,9 @@ class PinPasswordActivityPresenter @Inject constructor(
       lifecycleOwner = activity
       viewModel = pinViewModel
     }
-
+    binding.inputPin.setPasswordHidden(!pinViewModel.showPassword.get()!!)
```

Comment:
try to do it with data binding, in layout file only by setting `showPassword` to this newly introduced passwordHidden method.

Reply:
I don't get you, isn't the value in the pinViewModel.showPassword already retrieved from data binding in the xml file?

Reply:
Please refer this reference - https://developer.android.com/topic/libraries/data-binding/expressions

Reply:
Please don't resolve these comments, the author needs to resolve such comments.

Reply:
@anandwana001 So are you asking me to invoke the method directly inside the XML file and not when in the onClickListener block?

Reply:
Yes, not kotlin work, just the data binding in xml file will work.

Reply:
I can't call that method in the xml file, please help

Reply:
`app:passwordHidden="@{viewmodel.showPassword}"` That's all.

Reply:
I didn't know there was such an attribute for the pinView, thank you.